make/lib/Awt2dLibraries.gmk

```diff
@@ -555,24 +555,40 @@ else
   ifeq ($(call isTargetOs, solaris), true)
     HARFBUZZ_CFLAGS += -DHAVE_SOLARIS_ATOMIC_OPS
   endif
-  # hb-ft.cc is not presently needed, and requires freetype 2.4.2 or later.
-  LIBFONTMANAGER_EXCLUDE_FILES += libharfbuzz/hb-ft.cc
+  
+   # Early re-canonizing has to be disabled to workaround an internal XlC compiler error
+   # when building libharfbuzz
+   ifeq ($(call isTargetOs, aix), true)
+     HARFBUZZ_CFLAGS += -qdebug=necan
+   endif
```

Comment:
This is actually a change from [JDK-8258484](https://bugs.openjdk.org/browse/JDK-8258484) which is not in JDK 11u. Intentional? If yes, we should probably use `/issue add JDK-8258484` to have it tracked.

Reply:
Yep. this is intentional. Will add the issue.

Reply:
Why are you including this? Have you built this on AIX?